libraries/aws_security_groups.rb

```diff
@@ -44,11 +45,20 @@ def fetch_data
           group_id: security_group.group_id,
                                     vpc_id: security_group.vpc_id,
                                     group_name: security_group.group_name,
+                                    tags: fetch_tags(security_group.tags),
         }]
       end
       break unless @api_response.next_token
       pagination_options = { next_token: @api_response.next_token }
     end
     @table = security_group_rows
   end
+
+  def fetch_tags(tag_list)
+    tags = {}
+    tag_list.each do |tag|
+      tags[tag[:key].to_sym] = tag[:value]
```

Comment:
I posted above about changing this to a `flat_map`. Better yet it could be a flat map method in the base class - since this code will inevitably be repeated dozens of times accrosss resources.